apps/dot-voting/app/App.js

```diff
@@ -88,7 +88,7 @@ const App = () => {
     votes = [],
     entries = [],
     voteTime = 0,
-    minParticipationPct = 0,
+    globalMinQuorum = 0,
     pctBase = 0,
   } = appState
 
```

Comment:
I am not sure if the function below makes sense now that AddressBook integration is removed, we might want clean out in case this is not needed